protocol/messenger_activity_center.go

```diff
@@ -447,6 +447,14 @@ func (m *Messenger) DismissAllActivityCenterNotificationsFromUser(ctx context.Co
 	return notifications, m.syncActivityCenterDismissed(ctx, notifications, updatedAt)
 }
 
+func (m *Messenger) DismissActivityCenterNotificationsCommunity(ctx context.Context, communityID string, updatedAt uint64) ([]*ActivityCenterNotification, error) {
```

Comment:
rename `DismissActivityCenterNotificationsCommunity` to `DismissActivityCenterNotificationsByCommunity` sounds better?